SpywareFilter/sections/tracking_servers_firstparty.txt

```diff
@@ -26,9 +26,12 @@
 ! Disabled in DNS filter
 ||telemetry.mozilla.org^
 !
+||eventapi.afreecatv.com^
 ||stats.muasamcong.gov.vn^
 ||beacon.watch.impress.co.jp^
 ||media.fc2.com^$image
+||stat.nate.com^
+||hitweb2.chosun.com^
```

Comment:
not reproduced on chosun.com

Reply:
![Screenshot](https://user-images.githubusercontent.com/37463155/109114525-b21ab980-7735-11eb-8acd-9818b685dec0.png)
![Screenshot](https://user-images.githubusercontent.com/37463155/109114615-c78fe380-7735-11eb-872b-472950d6ce93.png)
https://realty.chosun.com/
https://realty.chosun.com/site/data/html_dir/2021/02/24/2021022402779.html